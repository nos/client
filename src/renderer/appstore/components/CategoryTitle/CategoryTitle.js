import React from 'react';
import { node } from 'prop-types';

import styles from './CategoryTitle.scss';

const CategoryTitle = ({ children }) => <h2 className={styles.categoryTitle}>{children}</h2>;

CategoryTitle.propTypes = {
  children: node.isRequired
};

export default CategoryTitle;
