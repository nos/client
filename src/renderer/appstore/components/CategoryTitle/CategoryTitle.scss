h2.categoryTitle {
  @extend %labelColor;
  margin: 0 0 12px;
  font-size: 28px;
  color: $gray-5;
  text-transform: capitalize;
  font-weight: normal;
}
