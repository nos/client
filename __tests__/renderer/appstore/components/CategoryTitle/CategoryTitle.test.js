import React from 'react';
import { mount } from 'enzyme';

import CategoryTitle from 'appstore/components/CategoryTitle/CategoryTitle';

describe('<CategoryTitle />', () => {
  it('renders correctly', () => {
    const wrapper = mount(<CategoryTitle>TestTitle</CategoryTitle>);
    expect(wrapper).toMatchSnapshot();
  });
});
