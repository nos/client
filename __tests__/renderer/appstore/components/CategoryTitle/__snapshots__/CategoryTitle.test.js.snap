// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CategoryTitle /> renders correctly 1`] = `
ReactWrapper {
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "getWrappingComponentRenderer": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
    "wrapInvoke": [Function],
  },
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "children": "TestTitle",
    },
    "ref": null,
    "rendered": Object {
      "instance": <h2
        class="categoryTitle"
      >
        TestTitle
      </h2>,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "TestTitle",
        "className": "categoryTitle",
      },
      "ref": null,
      "rendered": Array [
        "TestTitle",
      ],
      "type": "h2",
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": "TestTitle",
      },
      "ref": null,
      "rendered": Object {
        "instance": <h2
          class="categoryTitle"
        >
          TestTitle
        </h2>,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "TestTitle",
          "className": "categoryTitle",
        },
        "ref": null,
        "rendered": Array [
          "TestTitle",
        ],
        "type": "h2",
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromError": true,
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
  Symbol(enzyme.__linkedRoots__): Array [],
  Symbol(enzyme.__unrendered__): <CategoryTitle>
    TestTitle
  </CategoryTitle>,
  Symbol(enzyme.__updatedBy__): null,
}
`;
